docs/docs/users/reference/metrics.md

```diff
@@ -16,7 +16,11 @@ title: Metrics
 | `peer_failure_total`           | Counter   | Count   | Total number of failed peer requests                                                         |
 | `full_peers`                   | Gauge     | Count   | Number of healthy peers recognized by the node                                               |
 | `bad_peers`                    | Gauge     | Count   | Number of bad peers recognized by the node                                                   |
+| `network_height`               | Gauge     | Count   | The current network height                                                                   |
```

Comment:
It'd be good to explain the difference between those metrics in the docs (dedicated section).

```
head_epoch 3021420
network_height 3021421
expected_network_height 3023095
```

Reply:
@LesnyRumcajs The metrics are the same, and the difference is due to how the value is updated. I can change  `head_epoch` to the collect approach.